libs/librrgraph/src/io/rr_graph_reader.cpp

```diff
@@ -115,3 +133,85 @@ void load_rr_file(RRGraphBuilder* rr_graph_builder,
             read_rr_graph_name);
     }
 }
+
+static RREdgeId process_rr_edge_override(const std::string& line,
+                                         float& overridden_Tdel,
+                                         const RRGraphView& rr_graph) {
+    std::istringstream iss(line);
+    char ch;
+    RREdgeId edge_id;
+
+    if (std::isdigit(line[0])) {
+        // Line starts with an integer
+        int first;
+        iss >> first;
+        edge_id = (RREdgeId)first;
```

Comment:
Try all sorts of broken input, make sure you get an error message.
Ideally, you'd accept comment characters of # to the end of the line, as our other file formats do.